app/models/UserAnswers.scala

```diff
@@ -29,7 +29,8 @@ final case class UserAnswers(
   data: JsObject = Json.obj(),
   tasks: Map[String, TaskStatus] = Map(),
   status: SubmissionState.Value,
-  departureId: Option[String] = None
+  departureId: Option[String] = None,
+  isTransitional: Boolean = false
```

Comment:
We then wouldn't need to add isTransitional to our frontend user answers if it is only used in the API